build.gradle

```diff
@@ -106,6 +107,28 @@ allprojects {
         it.dependsOn = []
     }
 
+    findbugs {
+        sourceSets = [sourceSets.main]
+        excludeFilter = new File(rootDir, "findBugsSuppressions.xml")
+        effort = "max"
+    }
+
+    task checkFindBugsReport << {
+        try {
+            def xmlReport = findbugsMain.reports.getXml()
+            def slurped = new XmlSlurper().parse(xmlReport.destination)
+            def bugsFound = slurped.BugInstance.size()
+            if (bugsFound > 0) {
+                new XmlNodePrinter().printList(slurped.BugInstance.list())
```

Comment:
@AustinShalit @JLLeitschuh what's the problem? If it's not printing, it's likely because, to my knowledge, XmlNodePrinter requires an output (try passing System.out as a constructor argument)